---
description: "Obtenga más información sobre: no se ha alcanzado nunca el bloque ' Catch '; se <exception> controló anteriormente en la misma instrucción ' Try '"
title: No se alcanzó el bloque 'Catch'; '<exception>' se controla anteriormente en la misma instrucción Try
ms.date: 07/20/2015
f1_keywords:
- bc42031
- vbc42031
helpviewer_keywords:
- BC42031
ms.assetid: 7d15597c-5988-42ea-a853-63cbf78faaf3
ms.openlocfilehash: fc52c21f7a319c26b6ca7128c8d2d781db1a58c1
ms.sourcegitcommit: 10e719780594efc781b15295e499c66f316068b8
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 02/14/2021
ms.locfileid: "100473026"
---
# <a name="catch-block-never-reached-exception-handled-above-in-the-same-try-statement"></a><span data-ttu-id="777c8-103">No se alcanzó el bloque 'Catch'; '\<exception>' se controla anteriormente en la misma instrucción Try</span><span class="sxs-lookup"><span data-stu-id="777c8-103">'Catch' block never reached; \<exception> handled above in the same 'Try' statement</span></span>

<span data-ttu-id="777c8-104">Un bloque `Catch` del código no se puede alcanzar porque se controla en un bloque `Try` precedente.</span><span class="sxs-lookup"><span data-stu-id="777c8-104">A `Catch` block in the code cannot be reached because it is handled in a preceding `Try` block.</span></span>  
  
<span data-ttu-id="777c8-105">De forma predeterminada, este mensaje es una advertencia.</span><span class="sxs-lookup"><span data-stu-id="777c8-105">By default, this message is a warning.</span></span> <span data-ttu-id="777c8-106">Para más información sobre cómo ocultar las advertencias o cómo tratarlas como errores, vea [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).</span><span class="sxs-lookup"><span data-stu-id="777c8-106">For more information about hiding warnings or treating warnings as errors, see [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).</span></span>
  
 <span data-ttu-id="777c8-107">**Identificador de error:** BC42031</span><span class="sxs-lookup"><span data-stu-id="777c8-107">**Error ID:** BC42031</span></span>  
  
## <a name="to-correct-this-error"></a><span data-ttu-id="777c8-108">Para corregir este error</span><span class="sxs-lookup"><span data-stu-id="777c8-108">To correct this error</span></span>  
  
- <span data-ttu-id="777c8-109">Quite la instrucción redundante.</span><span class="sxs-lookup"><span data-stu-id="777c8-109">Remove the redundant statement.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="777c8-110">Consulte también</span><span class="sxs-lookup"><span data-stu-id="777c8-110">See also</span></span>

- [<span data-ttu-id="777c8-111">Cómo usar el bloque Try/Catch para detectar excepciones</span><span class="sxs-lookup"><span data-stu-id="777c8-111">How to use the try/catch block to catch exceptions</span></span>](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [<span data-ttu-id="777c8-112">Utilizar controladores de excepciones filtradas por el usuario</span><span class="sxs-lookup"><span data-stu-id="777c8-112">Using User-Filtered Exception Handlers</span></span>](../../standard/exceptions/using-user-filtered-exception-handlers.md)
- [<span data-ttu-id="777c8-113">Try...Catch...Finally (instrucción)</span><span class="sxs-lookup"><span data-stu-id="777c8-113">Try...Catch...Finally Statement</span></span>](../language-reference/statements/try-catch-finally-statement.md)
